docs/uitour.rst

```diff
@@ -48,6 +48,10 @@ Note that ``browser.uitour.testingOrigins`` can be a comma separated list of dom
     must also exclude the domain protocol e.g. ``https://``. A browser restart is also required
     after adding an allowed domain.
 
+    If you are working on Firefox Accounts integration, you can use the ``identity.fxaccounts.autoconfig.uri``
```

Comment:
I've tried setting this to both `https://accounts.firefox.com.cn/` and `https://accounts.stage.mozaws.net/`, but I can't seem to get UITour to notice the change. Is there anything else I need to do, other than restart Firefox?

Reply:
I've also tried setting `Services.prefs.getDefaultBranch("distribution.").setCharPref("id", "MozillaOnline");` in the browser console, to make Firefox act like the China Repack, but clicking the links I'm still seeing UITour open `https://accounts.firefox.com/`

Reply:
![image](https://user-images.githubusercontent.com/128755/87180479-9b127e80-c2ae-11ea-9604-defaf2e32fff.png)

@alexgibson could you check if your `identity.fxaccounts.remote.root` updated when you changed the autoconfig uri? 
I tried it locally in Nightly and it was not set, but after I updated and restarted it started working.

It is set in https://searchfox.org/mozilla-central/source/services/fxaccounts/FxAccountsConfig.jsm#282

Reply:
Yeah this seems to be the problem: my `identity.fxaccounts.remote.root` is still set to `https://accounts.firefox.com/`.

I can't seem to get it to change at all though. I've tried updating Nightly and restarting the browser, but no luck?

Reply:
Ok, I was able to get this to work when trying our a new profile in Nightly, so I can confirm that the China Repack support still works as expected. No idea what's wrong with my regular profile still, but at least it seems to be related to my install.

Reply:
Yea I was wondering if I should add that to the doc, it could be an issue in Firefox setting multiple config prefs in a row?? 😕 . I will add a note to the doc just so it is explained. I am happy you got it to work!